components/Player/PlayerHeader.css

```diff
@@ -24,10 +24,6 @@
   composes: image;
```

Comment:
I'm so confused why this thing keeps adding that. I'll just do the rmrf before each push now I guess